src/org/opendatakit/briefcase/ui/export/components/ConfigurationPanelForm.java

```diff
@@ -119,6 +124,7 @@ protected ConfigurationPanelForm(ConfigurationPanelMode mode) {
     });
     pullBeforeField.addActionListener(__ -> triggerChangePullBefore());
     pullBeforeOverrideField.addActionListener(__ -> triggerChangePullBeforeOverride());
+    overwriteExistingFilesCheckBox.addActionListener(__ -> triggerOverwriteExistingFiles());
```

Comment:
The expected behavior of this checkbox is:
1. Ask for confirmation of this action by the user and 
2. Trigger the change if we get the confirmation

Reading this line, it's like we always trigger the change (no traces of the confirmation request here). If the confirmation request happens inside `triggerOverwriteExistingFiles()` method, then its name should be `requestConfirmationAndOverwriteExistingFilesIfTheUserConfirms()`, which is more accurate but painfully long and weirdly specific. This is a code-smell telling us that the `requestConfirmationAndOverwriteExistingFilesIfTheUserConfirms()` method does too much stuff and it should be simplified. The best function is the one that does just one thing, and this one does three things.

Instead of renaming the method I'd suggest to be more explicit and write this listener:
```java 
overwriteExistingFilesCheckBox.addActionListener(__ -> {
  if (requestOverwriteExistingFilesConfirmation())
    triggerOverwriteExistingFiles();
});
```

This way, the confirmation is explicitly expressed inside the action listener and it's easy to understand that we only trigger the change if the user confirms the action.